src/components/Comments/ReplyCommentItem.js

```diff
@@ -59,6 +62,7 @@ const ReplyCommentItem = ({
     loadingDm: false,
     loadingDmAnon: false
   });
+  const [userAllowDm, setUserAllowDm] = React.useState(false);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The state variable `userAllowDm` has been introduced. Make sure to handle its initial state properly and update it as needed throughout the component lifecycle.



<!-- This is an auto-generated comment by OSS CodeRabbit -->